README.md

```diff
@@ -273,6 +274,23 @@ var logger = require('pino-http')({
 })
 ```
 
+If you prefer to work with the raw value directly, you can pass in `opts.wrapSerializers` as `false`:
+
+```js
+var http = require('http')
+var logger = require('pino-http')({
+  wrapSerializers: false,
+  serializers: {
+    req (req) {
+      // `req` is the raw `IncomingMessage` object, not the already serialized request from `pino.stdSerializers.req`.
+      return {
+        message: req.foo
+      };
+    }
+  }
+})
+```
+
```

Comment:
I wasn't quite sure of the best name for the option, some ideas were:

wrapSerializers: false
preSerialize: false
raw: true
serializeRaw: true

Reply:
`wrapSerializers` is ok.